api_tests/tests/dry/peers_using_ip.ts

```diff
@@ -14,19 +14,27 @@ import { Actor } from "../../src/actors/actor";
 async function assertNoPeersAvailable(actor: Actor, message: string) {
     const peersResponse = await request(actor.cndHttpApiUrl()).get("/peers");
 
-    expect(peersResponse.status).to.equal(200);
-    expect(peersResponse.body.peers, message).to.be.empty;
+    expect(peersResponse.status).toEqual(200);
+    try {
+        expect(peersResponse.body.peers).toBeArrayOfSize(0);
+    } catch (e) {
+        throw new Error(message);
```

Comment:
This kind of evolution of code is why I often prefer to not push changes like the one requested in https://github.com/comit-network/comit-rs/pull/1978 too much down the line :(

cc @bonomat: https://github.com/comit-network/comit-rs/issues/2013

They just don't get addressed and things get progressively worse from there ..